src/main/java/com/conveyal/r5/analyst/cluster/GridResultAssembler.java

```diff
@@ -135,17 +132,17 @@ public void handleMessage (Message message) {
             Origin origin = Origin.read(bais);
 
             // if this is not the first request, make sure that we have the correct number of iterations
-            if (buffer != null && origin.accessibilityPerIteration.length != this.nIterations) {
+            if (buffer != null && origin.samples.length != this.nIterations) {
```

Comment:
This updated version is fine since it provides more detail, I was more concerned about the word being used alone.